lockfile.json

```diff
@@ -1,188 +1 @@
4d0db15ae6f9f49cf2e759f06dba"}, "tsx": {"revision": "28e757a2f498486931b3cb13a100a1bcc9261456"}, "supercollider": {"revision": "bb18cc5e3278cd8baa81a34a56a4215dfb8c8425"}, "typescript": {"revision": "28e757a2f498486931b3cb13a100a1bcc9261456"}, "regex": {"revision": "3041aa3472d16fd94c6a9e15b741dbfecd9b714e"}, "fish": {"revision": "1c25205cd7a68434bd1e576972193b3994404307"}, "devicetree": {"revision": "fa70098cd70393f84785f85cdc6a45299b59cd5b"}, "c": {"revision": "008008e30a81849fca0c79291e2b480855e0e02c"}, "verilog": {"revision": "1b624ab8b3f8d54ecc37847aa04512844f0226ac"}, "php": {"revision": "6a271f4075e11815e787df9055a950fb844ee63b"}, "fennel": {"revision": "bc689e2ef264e2cba499cfdcd16194e8f5fe87d2"}, "teal": {"revision": "1f9562185ef09863a3eee5f5f1ed2858b94c8cda"}, "rst": {"revision": "5aee336764698c848160e495277bd47184d94339"}, "dockerfile": {"revision": "b25f86c3b2ec4192d040432a1a279861df390762"}, "html": {"revision": "d93af487cc75120c89257195e6be46c999c6ba18"}, "cpp": {"revision": "c61212414a3e95b5f7507f98e83de1d638044adc"}, "jsdoc": {"revision": "189a6a4829beb9cdbe837260653b4a3dfb0cc3db"}, "vue": {"revision": "91fe2754796cd8fba5f229505a23fa08f3546c06"}, "glimmer": {"revision": "87924bda53d0613b09d8f3004c17f6b0be5b6176"}, "gomod": {"revision": "2dfd6b7d3a4cfedc52e5eaa3cc497f419e628ebb"}, "bibtex": {"revision": "ccfd77db0ed799b6c22c214fe9d2937f47bc8b34"}, "elm": {"revision": "764876b93ab9d57532b6c550883b027a986a14db"}, "toml": {"revision": "8bd2056818b21860e3d756b5a58c4f6e05fb744e"}, "r": {"revision": "9a7f8d6429a48116f4c757ec0abbe42ac87c1f8e"}, "c_sharp": {"revision": "d5b3e64752426c9f15b4a407790ca362b7c86bc3"}, "python": {"revision": "d6210ceab11e8d812d4ab59c07c81458ec6e5184"}, "json": {"revision": "65bceef69c3b0f24c0b19ce67d79f57c96e90fcb"}, "css": {"revision": "94e10230939e702b4fa3fa2cb5c3bc7173b95d07"}, "ocamllex": {"revision": "ac1d5957e719d49bd6acd27439b79843e4daf8ed"}, "latex": {"revision": "7f720661de5316c0f8fee956526d4002fa1086d8"}, "commonlisp": {"revision": "48189d1627bdf9795b4c4d970169ec1d143ac6b9"}, "haskell": {"revision": "2e33ffa3313830faa325fe25ebc3769896b3a68b"}, "yaml": {"revision": "0e36bed171768908f331ff7dff9d956bae016efb"}, "ocaml": {"revision": "0348562f385bc2bd67ecf181425e1afd6d454192"}, "erlang": {"revision": "fea0141fb1e6d3af45c3d8f45441766793ed6e30"}, "ledger": {"revision": "28f79fd3ff79e69be18f46e468e8459e44fa901d"}, "graphql": {"revision": "5e66e961eee421786bdda8495ed1db045e06b5fe"}, "julia": {"revision": "12ea597262125fc22fd2e91aa953ac69b19c26ca"}, "kotlin": {"revision": "48bfb38abd93f8d617877d7bb7f92a6bb1166285"}, "turtle": {"revision": "05b2419c35ca64ad7a51dc5620426d5c9bf9c357"}, "rust": {"revision": "a360da0a29a19c281d08295a35ecd0544d2da211"}, "swift": {"revision": "a22fa5e19bae50098e2252ea96cba3aba43f4c58"}, "dart": {"revision": "6f332a211afa19920c851456d587d6c65f243832"}, "fortran": {"revision": "6d69c865d5c0a8a583a4e82a5ef75f8f5faab85b"}, "sparql": {"revision": "cf57a922b6a48664131617a5c7fe06e401e700f7"}}
```

Comment:
This file is autogenerated. Please don't touch it

Reply:
Do you have any tips on resolving this? I am not sure how to approach it

Reply:
you can use git checkout to recover a previous version, or you can also just copy the content from the current master branch https://github.com/nvim-treesitter/nvim-treesitter/blob/master/lockfile.json

Reply:
I've copied the latest lockfile to this branch now. Sorry for the mistake

Reply:
Looks like the file still has edits, you can try

```
git checkout master
git pull origin master
git checkout typecheck-in-local-nodes
git checkout master lockfile.json
git add lockfile.json
```

Reply:
Thanks - tried it now and it looks good